pycbc/strain/strain.py

```diff
@@ -508,29 +508,35 @@ def insert_strain_option_group(parser, gps_times=True):
 
     # Injection options
     data_reading_group.add_argument("--injection-file", type=str,
-                      help="(optional) Injection file used to add "
-                           "waveforms into the strain")
+                      help="(optional) Injection file containing parameters"
+                           " of CBC signals to be added to the strain")
     data_reading_group.add_argument("--sgburst-injection-file", type=str,
-                      help="(optional) Injection file used to add "
-                      "sine-Gaussian burst waveforms into the strain")
+                      help="(optional) Injection file containing parameters"
+                      "of sine-Gaussian burst signals to add to the strain")
     data_reading_group.add_argument("--injection-scale-factor", type=float,
-                    default=1, help="Divide injections by this factor "
-                    "before injecting into the data.")
+                      default=1,
+                      help="Divide injections by this factor "
+                           "before adding to the strain data")
     data_reading_group.add_argument("--injection-sample-rate", type=float,
-                    help="Sample rate for injections")
-    data_reading_group.add_argument('--injection-f-ref', type=float,
-                                    help='Reference frequency in Hz for '
-                                         'creating CBC injections from an XML '
-                                         'file.')
-    data_reading_group.add_argument('--injection-f-final', type=float,
-                                    help='Override the f_final field of a CBC '
-                                         'XML injection file.')
+                      help="Sample rate to use for injections (integer Hz). "
+                           "Typically similar to the strain data sample rate."
+                           "If not provided, the strain sample rate will be "
+                           "used")
+    data_reading_group.add_argument("--injection-f-ref", type=float,
+                      help="Reference frequency in Hz for creating CBC "
+                           "injections from an XML file")
+    data_reading_group.add_argument("--injection-f-final", type=float,
+                      help="Sample rate to use for injections (integer Hz). "
+                           "Typically similar to the strain data sample rate."
+                           "If not provided, the strain sample rate will be "
+                           "used")ide the f_final field of a CBC XML "
+                           "injection file (frequency in Hz)")
```

Comment:
Huh?

Reply:
yes, the tests didn't like this copy paste typo either